geoai/foundation_models.py

```
@@ -0,0 +1,704 @@
+    Returns:
+        True if ``terratorch`` can be imported, False otherwise.
+
+    Example:
+        >>> import geoai
+        >>> if geoai.check_terratorch_available():
+        ...     model = geoai.load_foundation_model("prithvi-eo-2.0-300m")
+    """
+    try:
+        import terratorch.models.backbones  # noqa: F401
+
+        return True
```

Ruff (0.15.15)

[warning] Consider moving this statement to an `else` block

(TRY300)